.ci/magic-modules/create-pr.sh

```diff
@@ -34,22 +36,32 @@ if [ "$BRANCH_NAME" = "$ORIGINAL_PR_BRANCH" ]; then
   # There is no existing PR - this is the first pass through the pipeline and
   # we will need to create a PR using 'hub'.
   if [ -n "$TERRAFORM_REPO_USER" ]; then
-    pushd build/terraform
-
-    git log -1 --pretty=%B > ./downstream_body
-    echo "" >> ./downstream_body
-    echo "<!-- This change is generated by MagicModules. -->" >> ./downstream_body
-    if [ -n "$ORIGINAL_PR_USER" ]; then
-      echo "/cc @$ORIGINAL_PR_USER" >> ./downstream_body
-    fi
-
-    git checkout -b "$BRANCH_NAME"
-    if TF_PR=$(hub pull-request -b "$TERRAFORM_REPO_USER/terraform-provider-google:2.0.0" -F ./downstream_body); then
-      DEPENDENCIES="${DEPENDENCIES}depends: $TF_PR ${NEWLINE}"
-    else
-      echo "Terraform - did not generate a PR."
-    fi
-    popd
+    for VERSION in "${TERRAFORM_VERSIONS[@]}"; do
```

Comment:
I still kind of think this is a sort of false generality - the best version of generality would be a lookup from version string to provider name.  I also kind of think that it's odd to special-case one out of two values here.

Reply:
just to confirm what this would mean in practice- having a map variable that we can do the PROVIDER_NAME/SUBMODULE_DIR/etc lookups in?

Reply:
That's the best implementation I can imagine, yeah.  As-is, I argue we're very likely to need to change it up if we ever add an tpg-alpha, and a lookup gives us flexibility.

Reply:
This is great!

Reply:
Thanks!